source/sessions/driver-sessions.rst

```diff
@@ -1269,3 +1310,4 @@ Change log
 :2021-04-08: Updated to use hello and legacy hello
 :2021-04-08: Adding in behaviour for load balancer mode.
 :2020-05-26: Simplify logic for determining sessions support
+:2022-01-19: Implicit sessions MUST obtain server session after connection checkout succeeds
```

Comment:
Leaving this comment open to make sure we update the date before merging, but on the same note, should there be a minor version bump to the spec?

Reply:
I a version bump makes sense, this is a behavioral change to the spec. updated!